chemicalx/models/gcnbmp.py

```diff
@@ -37,10 +38,10 @@ def circular_correlation(left_x, right_x):
 
 
 class Highway(nn.Module):
-    def __init__(self, input_size, prev_input_size):
+    def __init__(self, input_size: int, prev_input_size: int):
         """Instantiate the Highway update layer based on https://arxiv.org/abs/1505.00387
-        :param input_size (int): current representation.
-        :param prev_input_size (int): representation obtained by the previous convolutional layer.
+        :param input_size (int): current representation size.
+        :param prev_input_size (int): size of the representation obtained by the previous convolutional layer.
```

Comment:
while you're adding the python type annotations, please take them out of the docstrings (they're not only redundant but also noisy when trying to read)

Reply:
Ok I was tryingto follow what I was seeing in some other models, will clean it now.